requirements.txt

```diff
@@ -0,0 +1,25 @@
+boto3==1.10.26
```

Comment:
Hey @shadowscatcher ,
In the interest of keeping the project lightweight, could you delete any and all libraries in this file that are not required? I don't think botocore is needed, google-auth, google-api-core, and a whole bunch of others.

Reply:
boto3 and gcp libraries are required by aws_push.py and gcp_push.py, and botocore and google-* are their dependencies. Here's a list of dependencies installed alongside boto3, for example:

```
botocore==1.15.14
docutils==0.15.2
jmespath==0.9.5
pkg-resources==0.0.0
python-dateutil==2.8.1
s3transfer==0.3.3
six==1.14.0
urllib3==1.25.8
```

In order to completely get rid of such dependencies, we have to write corresponding upload methods from scratch without official clients.

Reply:
Done. Protobuf is required by boto3.